### PR TITLE
wip export side PR

### PR DESCRIPTION
**Pull request by mixmix**
hey @soapdog here are the tweaks I've added to your branch in getting things working. I think the main thing that worked was aggressivley getting sbot to connect to all peers.

Other changes to know about : 
- I modified my backup and filtered out all peers who had errors next to them. This left me with 36 I connected o at startup. Code here needs to reflect that
- I'm using https://github.com/ssbc/ssb-ebt/pull/11 (npm linked in) to get data about my identity seq on remote pubs. This bit is not yet linked in, but you can see the data being logged out.

Feels like it's coming together slowly. The Sync is pretty intense (it slowed my computer down massively and was hammering my internet!)

Will do some more on this tmrw,

### CODE REVIEW

**Comment by mixmix** [comment]
@soapdog I need to you review this and try it out please

File: app/html/app.js [hunk 1]
```diff
@@ -34,10 +36,31 @@ exports.create = (api) => {
 
       startApp()
 
+      onceTrue(api.sbot.obs.connection, server => getMySeq(server)) // EBT
+
       return app
     }
   })
 
+  function getMySeq (server) {
```
Comment (soapdog) [inline app/html/app.js]: Do you want to remove this before the merge?

**Comment by mixmix** [comment]
Does that app/html/app code still need removing?

**Comment by soapdog** [comment]
@mixmix I think you already removed it from branch `feature-key-export`, for me this is all good.

**Pull request merged**